src/csmacnz.Coveralls/Main.usage.txt

```diff
@@ -15,6 +15,7 @@ Options:
  --basePath <path>                        When useRelativePaths and a basePath is provided, this path is used instead of the current working directory.
  --opencover                              Reads input as OpenCover data.
  --dynamiccodecoverage                    Reads input as the CodeCoverage.exe xml format.
+ --exportcodecoverage					  Reads input as the Visual Studio Coverage Export xml format
```

Comment:
Sure, any other formatting?
On Mar 9, 2016 9:48 PM, "Mark Clearwater" <notifications@github.com> wrote:

> In src/csmacnz.Coveralls/Main.usage.txt
> <https://github.com/csMACnz/coveralls.net/pull/51#discussion_r55627555>:
>
> > @@ -15,6 +15,7 @@ Options:
> >   --basePath <path>                        When useRelativePaths and a basePath is provided, this path is used instead of the current working directory.
> >   --opencover                              Reads input as OpenCover data.
> >   --dynamiccodecoverage                    Reads input as the CodeCoverage.exe xml format.
> > + --exportcodecoverage					  Reads input as the Visual Studio Coverage Export xml format
>
> sorry to be pedantic, but could you use spaces not tabs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/csMACnz/coveralls.net/pull/51/files#r55627555>.
>